File: app/src/at-baole/java/asiantech/internship/summer/recyclerview/RecyclerViewActivity.java
package asiantech.internship.summer.recyclerview;

import android.os.Bundle;
import android.support.v4.app.FragmentTransaction;
import android.support.v7.app.AppCompatActivity;

import asiantech.internship.summer.R;
import asiantech.internship.summer.viewpager.TimelineFragment;

public class RecyclerViewActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_timeline);
        initView();
    }

    private void initView() {
        FragmentTransaction fragmentTransaction = getSupportFragmentManager().beginTransaction();
        fragmentTransaction.add(R.id.timelineFragment, new TimelineFragment());
        fragmentTransaction.commit();
    }
}

File: app/src/at-baole/java/asiantech/internship/summer/kotlin/recyclerview/RecyclerViewActivity.kt
package asiantech.internship.summer.kotlin.recyclerview

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import asiantech.internship.summer.R

class RecyclerViewActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.kotlin_activity_timeline)
        initView()
    }

    private fun initView(){
        val fragmentTransaction = supportFragmentManager.beginTransaction()
        fragmentTransaction.replace(R.id.timelineFragment, TimelineFragment())
        fragmentTransaction.commit()
    }
}
